po/nb-NO.po

```diff
@@ -0,0 +1,592 @@
+
+#: data/ui/dialogs/preferences.ui:174
+msgid "Stream public timelines"
+msgstr "Strøm offentlige tidslinjer"
+
+#: data/ui/dialogs/new_account.ui:21
+msgid "Add Account"
+msgstr "Legg til konto"
+
+#: data/ui/dialogs/new_account.ui:25 data/ui/views/base.ui:26
+#: src/Dialogs/NewAccount.vala:85
+msgid "Back"
+msgstr "Tilbake"
+
+#: data/ui/dialogs/new_account.ui:48 src/Dialogs/NewAccount.vala:84
+msgid "Next"
+msgstr "Neste"
+
+#: data/ui/dialogs/new_account.ui:129
+msgid "What is your instance?"
+msgstr "Hvilken instance bruker du?"
+
+#: data/ui/dialogs/new_account.ui:191
+msgid "<a href=\"https://joinmastodon.org/\">Don't have one yet?</a>"
+msgstr "<a href=\"https://joinmastodon.org/\">Har du ikke en enda?</a>"
+
+#: data/ui/dialogs/new_account.ui:242
+msgid "Enter authorization code"
+msgstr "Tast inn autorisasjonskode"
+
+#: data/ui/dialogs/new_account.ui:276
+msgid "<a href=\"tootle://manual_auth\">Try manual authorization</a>"
+msgstr "<a href=\"tootle://manual_auth\">Prøv manuell autorisering</a>"
+
+#: data/ui/widgets/accounts_button.ui:64 src/Views/Conversations.vala:6
+msgid "Conversations"
+msgstr "Samtaler"
+
+#: data/ui/widgets/accounts_button.ui:93 src/Views/Favorites.vala:6
+msgid "Favorites"
+msgstr "Favoritter"
+
+#: data/ui/widgets/accounts_button.ui:122 src/Views/Bookmarks.vala:6
+msgid "Bookmarks"
+msgstr "Bokmerker"
+
+#: data/ui/widgets/accounts_button.ui:152 src/Views/Lists.vala:59
+msgid "Lists"
+msgstr "Lister"
+
+#: data/ui/widgets/accounts_button.ui:204
+msgid "POSTS"
+msgstr "POSTER"
+
+#: data/ui/widgets/accounts_button.ui:247
+msgid "FOLLOWING"
+msgstr "FØLGER"
+
+#: data/ui/widgets/accounts_button.ui:290
+msgid "FOLLOWERS"
+msgstr "FØLGERE"
+
+#: data/ui/widgets/accounts_button.ui:344
+msgid "Refresh"
+msgstr "Last inn på nytt"
+
+#: data/ui/widgets/accounts_button.ui:357
+msgid "Preferences"
+msgstr "Instillinger"
+
+#: data/ui/widgets/accounts_button.ui:371
+msgid "About"
+msgstr "Om"
+
+#: data/ui/widgets/accounts_button.ui:397 src/Views/Search.vala:43
+msgid "Accounts"
+msgstr "Konti"
+
+#: data/ui/widgets/accounts_button_item.ui:60
+msgid "Forget"
+msgstr "Glem"
+
+#: data/ui/widgets/accounts_button_item.ui:88
+msgid "Title"
+msgstr "Tittel"
+
+#: data/ui/widgets/compose_attachment.ui:29
+msgid "Describe for the visually impaired"
+msgstr "Beskriv for de synshemmede"
+
+#: data/ui/widgets/compose_attachment.ui:90
+msgid "Unknown"
+msgstr "Ukjent"
+
+#: data/ui/widgets/compose_attachment.ui:113 data/ui/widgets/list_item.ui:74
+#: src/Widgets/Status.vala:243
+msgid "Delete"
+msgstr "Slett"
+
+#: data/ui/widgets/list_item.ui:26 src/Dialogs/ListEditor.vala:87
+msgid "Untitled"
+msgstr "Navnløs"
+
+#: data/ui/widgets/list_item.ui:42
+msgid "Edit"
+msgstr "Endre"
+
+#. Please do not remove the credits below. You may add your own, but keep the existing ones intact.
+#. TRANSLATORS: Replace this with your name. It will be displayed in the About dialog.
+#: src/Build.vala:15
+msgid " "
+msgstr ""
```

Comment:
Would you like to maintain this translation? If so, please insert your name here so it can be visible in the About dialog.

Reply:
Should be fixed now.